src/modules/function/invocation_utils.rs

```diff
@@ -67,7 +67,7 @@ fn run_function_with_args(meta: &mut ParserMetadata, mut fun: FunctionDecl, args
     })?;
     // Set the new return type or null if nothing was returned
     if let Type::Generic = fun.returns {
-        fun.returns = context.fun_ret_type.clone().unwrap_or_else(|| Type::Null);
+        fun.returns = context.fun_ret_type.clone().unwrap_or(Type::Null);
```

Comment:
This Clippy-suggested change has already been made on the master branch by @Ph0enixKM; but my change is identical, and should not cause a merge conflict.